### PR TITLE
Serialise `dt.time` objects to JSON

### PR DESCRIPTION
**Pull request by vthemelis**
Also added a unit test ran through ~~tox~~ pytest.

### CODE REVIEW

**Comment by vthemelis** [comment]
CC: @mariobuikhuizen

File: ipyaggrid/util.py [hunk 1]
```diff
@@ -44,12 +44,16 @@ def multi_options_to_json(value, widget):
...
+        if isinstance(obj, np.generic):
             return obj.item()
-        else:
-            raise TypeError(
-                "Unserializable object {} of type {}".format(obj, type(obj))
-            )
+
+        raise TypeError(
+            "Unserializable object {} of type {}".format(obj, type(obj))
+        )
```
Comment (vthemelis) [inline ipyaggrid/util.py:56]: Might even make sense to do:

```suggestion
        return str(obj)
```
?

**Comment by vthemelis** [comment]
@maartenbreddels , can I get an approval to run the workflows to see if this builds as expected?

**Comment by vthemelis** [comment]
Thanks! The build is 🟢

**Comment by vthemelis** [comment]
@maartenbreddels can I have a review on this? The build is green.

**Comment by maartenbreddels** [comment]
I think it looks good. The only thing is that we never use tox, so i'd rather see a simplified version that just runs pytest.

**Comment by vthemelis** [comment]
> I think it looks good. The only thing is that we never use tox, so i'd rather see a simplified version that just runs pytest.

@maartenbreddels , now just using plain `pytest`.

**Comment by maartenbreddels** [comment]
Great work Vasilis!

**Pull request merged**